Wiley Online Library.js

```diff
@@ -408,7 +408,7 @@ function scrapeBibTeX(doc, url) {
 					title: 'Full Text PDF',
 					mimeType: 'application/pdf'
 				});
-			}
+			}*/
```

Comment:
Please remove commented out code if the functionality is no longer in use.

Reply:
removed